lib/livebook/live_markdown/import.ex

```diff
@@ -390,10 +391,10 @@ defmodule Livebook.LiveMarkdown.Import do
         {Map.put(attrs, :default_language, default_language), messages}
 
       {"hub_id", hub_id}, {attrs, messages} ->
-        if Livebook.Hubs.hub_exists?(hub_id) do
-          {Map.put(attrs, :hub_id, hub_id), messages}
-        else
-          {attrs, messages ++ ["ignoring notebook Hub with unknown id"]}
+        cond do
+          Hubs.hub_exists?(hub_id) -> {Map.put(attrs, :hub_id, hub_id), messages}
+          Hubs.get_offline_hub(hub_id) -> {attrs, messages}
```

Comment:
We need to set the `:hub_id` here as well, no?

Reply:
No, we don't want the hub to leak. It is only used for verifying the stamp.